platform/android/android_input_handler.cpp

```diff
@@ -39,7 +39,7 @@ void AndroidInputHandler::process_joy_event(AndroidInputHandler::JoypadEvent p_e
 			Input::get_singleton()->joy_button(p_event.device, (JoyButton)p_event.index, p_event.pressed);
 			break;
 		case JOY_EVENT_AXIS:
-			Input::get_singleton()->joy_axis(p_event.device, (JoyAxis)p_event.index, p_event.value);
+			Input::get_singleton()->joy_axis(p_event.device, (JoyAxis)p_event.index, p_event.value, false);
```

Comment:
Warrants adding a comment to clarify that Android axis events are already normalized.

Is that true for all axes?

Reply:
After reading your comment, I did a bit of extra digging. It turns out, they're also already normalized on iOS. So I dug further and found that there are some Android trigger mappings that use the positive axis, and some that use the full range. I've updated the patch to take the controller mapping range into account before applying the correction. I don't have the hardware to test but I think my previous revision would have fixed some Android controllers (such as mine) and broken some others. Taking the specific device mapping into account sounds like a more robust solution.